docker/vars.opentest.sh

```diff
@@ -12,3 +12,5 @@ export GPC_CONSUMER_SPINE_CLIENT_CERT=""
 export GPC_CONSUMER_SPINE_CLIENT_KEY=""
 export GPC_CONSUMER_SPINE_ROOT_CA_CERT=""
 export GPC_CONSUMER_SPINE_SUB_CA_CERT=""
+
+export GPC_CONSUMER_SSP_FQDN = "https://proxy.opentest.hscic.gov.uk"
```

Comment:
No spaces around assignments in bash
```suggestion
export GPC_CONSUMER_SSP_FQDN="https://proxy.opentest.hscic.gov.uk"
```